mobile-api/Dockerfile

```
@@ -0,0 +1,12 @@
+FROM node:16-alpine AS builder
+WORKDIR /app
+COPY . .
+RUN npm ci
+RUN npm run build
+
```

```suggestion

FROM node:16-alpine AS dependencies
WORKDIR /build
COPY package-lock.json package.json ./
RUN npm ci --production

```
If we install the production dependencies in a build step, the final step can just copy them across.  This shaves a few MB off the final image.